subprojects/docs/src/samples/userguide/organizingGradleProjects/customGradleDistribution/build.gradle

```diff
@@ -0,0 +1,19 @@
+import org.gradle.distribution.DownloadGradle
```

Comment:
This is a really neat-o sample. I wonder if a project-based tutorial would be appropriate. WDYT?

Reply:
Can you explain what you mean by "project-based tutorial"? Do you mean writing a getting started guide? IMHO I don't think it's worth it to create a tutorial for it. 

Users can just copy-paste the example and be done with it. I believe there's no point in needing to explain how it works internally as long as it does work. I'd rather see this as an opportunity for a init type.

Reply:
Yes, a getting started tutorial is what I meant. 

We just have to make sure we link to the sample then so folks can copy it.

Reply:
As a user I'd feel more inclined to use something like this: `gradle init --type=custom-gradle-dist --gradle-version=4.6 --init-script-dir=/Users/bmuschko/enterprise-init-script`. I almost wouldn't have to think about it. It just works.